app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
```

Comment:
Is there a reason to have the `dissmiss-all-notices-button` class here? If there is one - why is it not applied below?

(just asking out of curiosity :thought_balloon: )